dev/integration_tests/android_host_app/app/src/main/java/io/flutter/add2app/MainActivity.java

```diff
@@ -5,7 +5,9 @@
 package io.flutter.add2app;
 
 import android.os.Bundle;
+
```

Comment:
Thanks for the PR. Did you use a tool to auto-format the Java files?

Reply:
Yes, Android studio. I did it using "Optimize Imports".